lib/slave/slave.go

```diff
@@ -11,27 +11,31 @@ import (
 
 var slaveInstance Slave
 
+type task struct {
+	lib.Task
+	Status       lib.Status
+	Password     string
+	progressChan chan chan string
+}
+
 type Slave struct {
+	id              string
```

Comment:
For now the identifier is always `instance.EC2.cong1`, but the idea was to give each slave an unique identifier?